tests/account_test.js

```diff
@@ -0,0 +1,24 @@
+const r_ship = Math.floor(Math.random() * 5) + 1
+
+Feature('Guestbook');
+    
+Scenario('I can verify that I am on the correct obit',  ({ I }) => {
+    //Given I am on a Janus obituary page
+    //When I visit Virginia's Obituary
+    I.amOnPage('/us/obituaries/chicagotribune/name/virginia-gruchalski-obituary?pid=196167379');
```

Comment:
You don't really need a // When comment here, and the //Given comment applies to `I.amOnPage` as it stands.  Feature name needs more description.